### PR TITLE
Add bash, buildbase, and librdkafka1.2.1

### PR DESCRIPTION
**Pull request by lexwraith**

### CODE REVIEW

File: Dockerfile.alpine [hunk 1]
```diff
@@ -32,6 +32,14 @@ RUN cd /tmp && \
 
 RUN update-ca-certificates
 
+# Librdkafka for event service dependents
+RUN curl -L https://github.com/edenhill/librdkafka/archive/v1.2.1.zip > librdkafka.zip && \
+  unzip librdkafka.zip && \
+  cd librdkafka-1.2.1 && \
+  ./configure --prefix /usr && \
+  make && \
+  make install
```
Comment (fluffywaffles) [inline Dockerfile.alpine:43]: `make install` with no prefix doesn't require superuser?

**Review by fluffywaffles — APPROVED** [review]

File: Dockerfile.alpine (hunk 1 above)
Comment (lexwraith) [reply Dockerfile.alpine:43]: apparently not. `sudo` was originally required as one of the additions to `apk install` but turns out it's unnecessary. Though I do wonder if that'll somehow affect the linking of the python package to `librdkafka.h`

Comment (lexwraith) [reply Dockerfile.alpine:43]: Nevermind, works fine 🤷‍♂

Comment (imsky) [reply Dockerfile.alpine:43]: this runs as root

File: Dockerfile.alpine [hunk 2]
```diff
@@ -11,7 +11,7 @@ RUN addgroup -g ${USER_ID} ${USER} && adduser -D -G ${USER} -u ${USER_ID} ${USER
 
 # System dependencies
 RUN apk add --no-cache \
-  linux-headers coreutils ca-certificates openssl openssl-dev musl-dev g++ python3 python3-dev git curl && \
+  linux-headers coreutils ca-certificates openssl openssl-dev musl-dev g++ python3 python3-dev git curl bash build-base && \
```
Comment (imsky) [inline Dockerfile.alpine:14]: what is bash used for?

File: Dockerfile.alpine [hunk 3]
```diff
@@ -32,6 +32,14 @@ RUN cd /tmp && \
 
 RUN update-ca-certificates
 
+# Librdkafka for event service dependents
+RUN curl -L https://github.com/edenhill/librdkafka/archive/v1.2.1.zip > librdkafka.zip && \
```
Comment (imsky) [inline Dockerfile.alpine:36]: should verify SHA checksum of this file

File: Dockerfile.alpine (hunk 1 above)
Comment (imsky) [inline Dockerfile.alpine:43]: should delete the kafka ZIP file after installation

File: Dockerfile.alpine (hunk 2 above)
Comment (lexwraith) [reply Dockerfile.alpine:14]: the make commands in source I believe

**Comment by erikbern** [comment]
no idea if it's a ton of work, but it might be worth building an alpine package that we can host for kafka

i did something similar with tensorflow (python package, but same principle): https://github.com/better/alpine-tensorflow

**Comment by lexwraith** [comment]
That could net us a few open source points. It should be pretty easy to do

**Comment by imsky** [comment]
:shipit:

**Comment by lexwraith** [comment]
thanks @imsky @fluffywaffles @erikbern

**Pull request merged**